vyper/semantics/analysis/imports.py

```diff
@@ -149,17 +152,19 @@ def _add_import(
             alias, qualified_module_name, compiler_input, ast
         )
 
-    # load an InterfaceT or ModuleInfo from an import.
-    # raises FileNotFoundError
-    def _load_import(self, node: vy_ast.VyperNode, level: int, module_str: str, alias: str) -> Any:
-        # the directory this (currently being analyzed) module is in
-        ast = self.graph.current_module
-        self_search_path = Path(ast.resolved_path).parent
+    def _load_file(self, path: PathLike, level: int):
+        if level == 0:
+            self.input_bundle.search_paths = self.absolute_search_paths
+        else:
+            ast = self.graph.current_module
+            current_search_path = Path(ast.resolved_path).parent
+            self.input_bundle.search_paths = [current_search_path]
 
-        with self.input_bundle.poke_search_path(self_search_path):
-            return self._load_import_helper(node, level, module_str, alias)
+        return self.input_bundle.load_file(path)
```

Comment:
nit: can we move this after `_load_import` to match the other functions, which are laid out in reverse dependency order